include/grpcpp/server_impl.h

```diff
@@ -182,7 +179,7 @@ class Server : public grpc::ServerInterface, private grpc::GrpcLibraryCodegen {
   ///
   /// \param sync_cq_timeout_msec The timeout to use when calling AsyncNext() on
   /// server completion queues passed via sync_server_cqs param.
-  Server(int max_message_size, ChannelArguments* args,
+  Server(ChannelArguments* args,
```

Comment:
I think this is clearly the right thing to do -- I'm not sure why this extra arg was ever here in the first place -- but this is technically a non-backward-compatible API change.  We can probably get away with this, because users should be using ServerBuilder instead of directly instantiating Server.  But let's get a review from @vjpai, just to make sure this shouldn't break anything for anyone.

Reply:
It's not API. The constructors of this class are private and only accessed by a friend. I have no concerns.